capitolweb/workers/text_utils.py

```diff
@@ -82,23 +82,23 @@ def remove_trailing_tokens(entity, verb=True, adposition=True, conjunction=True,
         # remove trailing verbs like don't and won't as well.
         if entity and len(entity) > 1:
             if entity[-1].tag_ == 'RB' and any(entity[-2].tag_ == v for v in VERB_TAGS):
-                logging.info('Removing trailing verb from named entity: %s', entity)
+                # logging.info('Removing trailing verb from named entity: %s', entity)
```

Comment:
Why are these commented out? They look like they'd be useful (albeit at the debug level, they are probably a bit too noisy for info).

Reply:
I agree; debug level is what i'd go with.

Reply:
My bad, I removed them because they created too much noise on the `info` level and I couldn't follow the segmentation. Put back in at `debug` level.